examples/source/interactivity-dynamic-content/Subscription_Settings/index.html

```diff
@@ -0,0 +1,434 @@
+    -->
+    <amp-list src="<% base_path %>/subscription" binding="refresh" items="." single-item layout="fill">
+      <template type="amp-mustache">
+        <form action-xhr="<% base_path %>/subscription" method="post" id="form1">
+          <select name="nextSubscription" on="change: form1.submit;">
+            {{#options}}
+              {{#isSelected}}
+                <option value="{{value}}" selected>
+                  {{text}}
+                </option>
+              {{/isSelected}}
+              {{^isSelected}}
+                <option value="{{value}}">
+                  {{text}}
+                </option>
+              {{/isSelected}}
+            {{/options}}
+          </select>
+        </form>
+      </template>
+      <div placeholder>Loading...</div>
+      <div fallback>Something went wrong. Please refresh</div>
+    </amp-list>
+
+    <!--
+      #### Step 4: Disabling While Submitting
+
+      After selecting a new subscription setting from the dropdown, it is unclear when the form is submitting or submitted. Furthermore, nothing prevents the user from attempting to select a new subscription setting while the form is still submitting.
+
+      Ideally the `select` element would be [disabled](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled) while the form is submitting. However, there's a problem with this approach: [values of disabled inputs are not submitted with a form](https://www.w3.org/TR/PR-html40-971107/interact/forms.html#:~:text=Values%20of%20disabled%20controls%20are%20not%20submitted%20with%20a%20form.). Under different circumstances we might consider using the [`readonly`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly) attribute instead of the `disabled` attribute to avoid this problem, but the [`select` element does not support the `readonly` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly#wikiArticle:~:text=The%20attribute%20is%20not%20supported%20or%20relevant%20to%20%3Cselect%3E%20or). Fortunately, we can solve this problem by submitting the `select` element's value through a new [hidden `input` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/hidden), which doesn't need to be disabled because users cannot interact with or see it, that we ensure always contains the same value as the `select` element.
+
+      We start by adding a hidden `input` element, and moving the `name` of the `select` element to the hidden `input` element so that its value is submitted. To keep the hidden `input` element's value in sync with the `select` element's value we update a new `nextSubscription` state variable using [`AMP.setState`](/documentation/components/amp-bind#updating-state-variables-with-amp.setstate()) whenever the `select` element's value changes and [bind](/documentation/components/amp-bind#bindings) the hidden `input` element's `value` attribute to the state variable.
```

Comment:
```suggestion
      We start by adding a hidden `input` element, and moving the `name` of the `select` element to the hidden `input` element so that its value submits. To keep the hidden `input` element's value in sync with the `select` element's value, we update a new `nextSubscription` state variable using [`AMP.setState`](/documentation/components/amp-bind#updating-state-variables-with-amp.setstate()) whenever the `select` element's value changes and [bind](/documentation/components/amp-bind#bindings) the hidden `input` element's `value` attribute to the state variable.
```